.github/workflows/Action.yml

```diff
@@ -0,0 +1,124 @@
+name: Build and Release
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]      
+
+jobs:
+  build:
+    
+    runs-on: windows-latest
+
+    steps: 
+    - uses: actions/checkout@v2.3.1
+    
+    - name: Fetch all history for all tags and branches
+      run: git fetch --prune --unshallow
+    - name: Install GitVersion
+      uses: gittools/actions/gitversion/setup@v0.9.3
+      with:
+          versionSpec: '5.3.x'
+    - name: Use GitVersion
+      id: gitversion # step id used as reference for output values
+      uses: gittools/actions/gitversion/execute@v0.9.3
+      with:
+        updateAssemblyInfo: true
+        updateAssemblyInfoFilename: ./cc-cli/AssemblyInfo.cs
+    - name: Print GitVersion computed
+      run: |
+        echo "LegacySemVer: ${{ steps.gitversion.outputs.legacySemVer }}"
+        echo "AssemblySemVer: ${{ steps.gitversion.outputs.assemblySemVer }}"
+    
+    
+    - name: Setup .NET Core SDK
+      uses: actions/setup-dotnet@v1.5.0
+      with:
+        dotnet-version: 2.1
+    
+    - name: Define Authentication
+      shell: bash
+      env:
+        CLIENT_ID: ${{ secrets.ClientID }}
+        
+      run: |
+        cat <<EOF > ./cc-cli/authconfig.json
+        {
+          "Authority": "https://auth0.hypertherm.com",
+          "Audience": "https://api.hypertherm.com/cutchart",
+
+          "### AUTH SECRETS ###": "### DO NOT COMMIT REAL VALUES BELOW THIS LINE ###",
```

Comment:
Do you still need this in the generated file? It might be a little cleaner to remove it.
```suggestion
```

Reply:
Yea it can be removed from the deployment pipeline, it is in the template to help remind us and others who pull the repo not to commit that info.